Header  header       # timestamp

float64 temperature  # temperature in celsius
float64 humidity     # humidity percentage

int64   light_level  # the level of illuminance of the ambience light, 1~20
